tests/models/test_tpu.py

```diff
@@ -216,7 +216,7 @@ def test_tpu_grad_norm(tmpdir):
     )
 
     model = BoringModel()
-    tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
+    trainer.fit(model)
```

Comment:
so is the accuracy test not desired here?

Reply:
Yeah, should have it actually. Reverted!